docs/tutorials/q32d14.ipynb

```diff
@@ -0,0 +1,3360 @@
+    "7/crIyPDeWnYkiVLtHbtWr300kt68sknNXDgwLD1kEHnM4g0r76+XocOHVJWVlbI+iOVkttFcwZd\n",
+    "3Q91NK+4uFgZGRnKzc11pn344YeqqqrSuXPnwl5kGM3jL3XvsaB///5qaGhw5gUCgZCz45WVlRo7\n",
+    "dmzYOsigaxnExsaqT58+8ng8mjlzpg4dOiRJevzxxyW1Pc43OG3hwoUqLy/X8ePHdfToUR09ejRk\n",
+    "O9GeAeyjgESxiRMnKikpSZs2bXJeOihJ48eP17Fjx2SM0cGDB53LGoKam5u1f//+kCdvBF+41rt3\n",
+    "bz366KPq0aOHM02S+vXrF7JTa25u1qZNm5SUlBR2yUo0iZRBV8ZfCv/pYnx8vNavX6+tW7eqqalJ\n",
+    "eXl5Yeshg85nEGleZWWlzp49q6lTp+qbb75RcXGxTp8+raqqKs2ZM0cffPCBSktLtXnz5pBtk0HX\n",
+    "9kPtzSspKdHZs2f1/vvvO8sG90MxMTHq06cP+6EIuutY0Lt3bwUCAV25ckVnzpwJudytrKws4k3P\n",
+    "ZNCmKxlcvXrV+b3f79ewYcPU1NTk/JsPTpMkj8ejuLg49ejRI6wokgFc4dbNJ/j/UFdXZz777DNT\n",
+    "WloactPZV199ZXJyckxubq6pqqoyCxYsMI2NjcYYY8rKysx7770Xsp6ioiLj8/nMxIkTTWlpqTHG\n",
+    "mNWrVxufz2dyc3PNm2++aVpaWowxxrS2tpr169ebzz//POIzyKNNpAw6O/4tLS0mNTXVNDc3O9O2\n",
+    "bt1q8vLyTH5+vikrKwtZngxCdTaD2+fdqri4OOQmdGPaHg5w+03oZPCvruyHbp8XCATMQw89ZDIz\n",
+    "M43P53MexlBcXGx8Pp/Jysoyn3zyibNuxj9Udx0LysvLzcSJE01mZqY5ePCgM33OnDnmjz/+CFmW\n",
+    "DEJ1NoOysjIzZswYk52dbV577TXT3Nxszp8/b8aMGWNycnLM5MmTTU1NjTHGmG3btpnnnnvOZGVl\n",
+    "mTfeeMNZPxnALR5jbnn4NKJSRUWFSktLNXr0aE2fPv2e3nxmjNGWLVt04MABzZo1S6mpqfdsW/cT\n",
+    "MnAfGbiL8XcfGbiPDBAtYj6NdFcZosqAAQPk9XpVXl6uS5cuadiwYYqJien27QRP8/7555+aMWOG\n",
+    "Ro4c2e3buF+RgfvIwF2Mv/vIwH1kgGhBAYEkaeDAgYqPj5ff79eRI0eUmJgor9fbbeuvqanRt99+\n",
+    "q9raWs2cOZOdXQRk4D4ycBfj7z4ycB8ZIBpwCRZCXLhwQZs2bdLZs2c1YcIE5ebm3tUTMa5fvy6/\n",
+    "36+9e/cqKSlJRUVFio+P78ZP/OAhA/eRgbsYf/eRgfvIAA8yCgjCtLa2as+ePdqxY4eMMUpNTdX4\n",
+    "8eP1xBNP3NH1qMYY1dbWav/+/aqoqJDH41F+fr4yMjJC3luB9pGB+8jAXYy/+8jAfWSABxUFBO1q\n",
+    "bGzUgQMH9Pvvv+vKlSvyer1KTEx0fvXq1Us9e/ZUIBDQjRs3dO7cOedXfX294uLiNG7cOKWnp/Nc\n",
+    "8S4iA/eRgbsYf/eRgfvIAA8aCgj+kzFGJ06c0MmTJ50d2q3v+AiKjY1VYmKiBg0apCFDhujpp5++\n",
+    "p0/wiCZk4D4ycBfj7z4ycB8Z4EFBAUGnGWNUX1+vmzdvKhAIqGfPnoqNjZXX62UHZwkZuI8M3MX4\n",
+    "u48M3EcGuF9RQAAAAABYwx1IAAAAAKyhgAAAAACwhgICAAAAwBoKCAAAAABrKCAAAAAArKGAAAAA\n",
+    "ALCGAgIAAADAGgoIAAAAAGsoIAAAAACsoYAAAAAAsIYCAgAAAMAaCggAAAAAayggAAAAAKyhgAAA\n",
+    "AACwhgICAAAAwBoKCAAAAABrKCAAAAAArKGAAAAAALCGAgIAAADAGgoIAAAAAGsoIAAAAACsoYAA\n",
+    "AAAAsIYCAgAAAMAaCggAAAAAayggAAAAAKyhgAAAAACwhgICAAAAwBoKCAAAAABrKCAAAAAArKGA\n",
+    "AAAAALCGAgIAAADAGgoIAAAAAGsoIAAAAACsoYAAAAAAsIYCAgAAAMAaCggAAAAAayggAAAAAKyh\n",
+    "gAAAAACwhgICAAAAwBoKCAAAAABrKCAAAAAArKGAAAAAALCGAgIAAADAGgoIAAAAAGsoIAAAAACs\n",
+    "oYAAAAAAsIYCAgAAAMAaCggAAAAAayggAAAAAKyhgAAAAACwhgICAAAAwBoKCAAAAABrKCAAAAAA\n",
+    "rKGAAAAAALCGAgIAAADAGgoIAAAAAGsoIAAAAACsoYAAAAAAsIYCAgAAAMAaCggAAAAAayggAAAA\n",
+    "AKyhgAAAAACwhgICAAAAwBoKCAAAAABrKCAAAAAArKGAAAAAALCGAgIAAADAGgoIAAAAAGsoIAAA\n",
+    "AACsoYAAAAAAsIYCAgAAAMAaCggAAAAAayggAAAAAKyhgAAAAACwhgICAAAAwBoKCAAAAABrKCAA\n",
+    "AAAArKGAAAAAALCGAgIAAADAmv8BR9AlDjpy8OwAAAAASUVORK5CYII=\n",
+    "\" alt=\"q32 grid\" />"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Building the circuit"
```

Comment:
Maybe add a little explanation here. As a tutorial, it's difficult to understand what this huge code cell is doing. I'll take your word for it that it's connecting up the circuit :)

Reply:
Added.